qtt/instrument_drivers/virtualAwg/sequencer.py

```diff
@@ -58,6 +58,32 @@ def make_square_wave(amplitude, period, repetitions=1, name='pulse'):
         return {'name': name, 'wave': SequencePT(*(sequence_data,)*repetitions),
                 'type': DataTypes.QU_PULSE}
 
+    @staticmethod
+    def make_pulse_table(amplitudes, waiting_times, repetitions=1, name='pulse_table'):
+        """ Creates a sequence of pulses from a list of amplitudes and waiting times.
+            Note that the initial voltage level will be given by the last element in amplitudes.
```

Comment:
This is really odd. Can we change such that the first value is the first value in the list of amplitudes?

Reply:
✔️